src/wire/Socket.ts

```diff
@@ -339,7 +341,8 @@ export class InteractiveSocket extends EventEmitter {
     }
 
     private sendRaw(packet: any) {
-        const data = JSON.stringify(packet);
+        // Replace circular references with nothing.
+        const data = stringify(packet, null, 0, () => {/** */});
```

Comment:
Some people are literally doing something like:
```
control.text='blah'
contro.progress = 1.0

client.updateControls({sceneID:'blah',controls:[control]);

```

Which leads to the circular as control has a ref to scene.

Should we instead add a `toJSON` to each tree area that strips out circulars?